aks/azure-ad-integration/azure-ad-integration.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+# The MIT License (MIT)
+#
+# Copyright (c) 2019 Microsoft Azure
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+# 
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+# 
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+# Define a variable for the AKS cluster name, resource group, and location
+aksname="myakscluster"
+resourcegroup="myResourceGroup"
+location="East US"
+
+# Create the Azure AD application
+serverApplicationId="$(az ad app create \
```

Comment:
Remove surrounding quotes; not necessary and causing problems with the other quotes embedded in the command. This must be done for all of the `$(...)` usage in this script.

Reply:
Removed surrounding quotes. Not sure why this was causing issues, it's been used extensively by PG and field teams for a while.